Xamarin.Forms.Core/VisualElement.cs

```diff
@@ -152,6 +152,46 @@ static void OnTransformChanged(BindableObject bindable, object oldValue, object
 
 		public static readonly BindableProperty BackgroundColorProperty = BindableProperty.Create("BackgroundColor", typeof(Color), typeof(VisualElement), Color.Default);
 
+		public static readonly BindableProperty BackgroundProperty = BindableProperty.Create(nameof(Background), typeof(Brush), typeof(VisualElement), Brush.Default,
+			propertyChanging: (bindable, oldvalue, newvalue) =>
+			{
+				if (oldvalue != null)
+					(bindable as VisualElement)?.StopNotifyingBackgroundChanges();
+			},
+			propertyChanged: (bindable, oldvalue, newvalue) =>
+			{
+				if (newvalue != null)
+					(bindable as VisualElement)?.NotifyBackgroundChanges();
+			});
+
+		void NotifyBackgroundChanges()
```

Comment:
Technically we need this everywhere we use brushes right? @StephaneDelcroix do you have any thoughts or any existing BO magic that we can use here so we don't have to do this every time there's a brush to use? 

Or should this be pushed down to the platforms to watch for brush changes?